search/lib/validator/index.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
+const yaml = require('js-yaml');
+const path = require('path');
+const Ajv = require('ajv');
+const ajv = new Ajv({ allErrors: true });
+
+const loadOpenApiYaml = (swaggerYaml) => {
+  if (!swaggerYaml) throw new Error('Missing Yaml path');
+  let yamlSchemaFile;
```

Comment:
Can we convert this to a ternary operator ( x ? y : z )?

Reply:
it can be moved to a single line.

`const yamlSchemaFile = path.isAbsolute(swaggerYaml) ? swaggerYaml : path.join(__dirname, swaggerYaml);`